easybuild/easyconfigs/n/NGSadmix/NGSadmix-32-GCC-7.3.0-2.30.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CmdCp'
+
+name = 'NGSadmix'
+version = '32'
+
+homepage = 'http://www.popgen.dk/software/index.php/NgsAdmix'
+description = """This page contains information about the program called 
+NGSadmix, which is a tool for finding admixture proportions 
+from NGS data, based on genotype likelihoods."""
+
+toolchain = {'name': 'GCC', 'version': '7.3.0-2.30'}
+
+# This is a bit of a hack as there is no easyblock to simply download a cpp file and compile it.
+# So we download the file from the source and copy it to the build directory. 
+
+sources = [{
+    'source_urls': ['http://popgen.dk/software/download/NGSadmix/'],
+    'filename': 'ngsadmix32.cpp',
+    'download_filename': 'ngsadmix32.cpp',
+    'extract_cmd': 'cp %s %(builddir)s',
+}]
+
+checksums = ['a5623b71d20114708cfd7ccc5f8c6ce38ff871d1b79af85bc89d38775f2d477e']
+
+build_str = "g++ ngsadmix32.cpp -O3 -lpthread -lz -o NGSadmix"
+cmds_map = [('', build_str)]
+
+files_to_copy = ['NGSadmix']
+
+sanity_check_paths = {
+    'files': ['NGSadmix'],
+    'dirs': [''],
```

Comment:
Please remove the empty string entry in `dirs`, doesn't make sense (you're checking for a non-empty installation directory this way, which is already done by checking for the `NGSadmix` file.

Reply:
Done